doc/source/whatsnew/v1.0.0.rst

```diff
@@ -456,6 +456,7 @@ Other
 - :meth:`DataFrame.to_csv` and :meth:`Series.to_csv` now support dicts as ``compression`` argument with key ``'method'`` being the compression method and others as additional compression options when the compression method is ``'zip'``. (:issue:`26023`)
 - Bug in :meth:`Series.diff` where a boolean series would incorrectly raise a ``TypeError`` (:issue:`17294`)
 - :meth:`Series.append` will no longer raise a ``TypeError`` when passed a tuple of ``Series`` (:issue:`28410`)
+- :meth:`Series.pct_change` will can handle duplicate axis (:issue:`28664`)
```

Comment:
move to reshaping, don't put things in Other

Reply:
Thank you for your information. I will be more careful in the future!